public/index.html

```diff
@@ -58,10 +58,10 @@ <h5 class="card-title">Business</h5>
                                 <div class="col-md-6" id="totalBusinessTask"></div>
                             </div>
                         </div>
-                    </div>
+                    </span>
                 </div>
                 <div class="col-md-4 col-12">
-                    <div class="card text-white mb-3" style="max-width: 18rem; background-color: #ef6c00;">
+                    <span id="Development" type="button" class="card text-white mb-3" style="max-width: 18rem; background-color: #ef6c00;">
```

Comment:
Please fix the span back to div this might create some problems with small screen, and type button is not needed it will work without that.